src/models/PlayerRanking.ts

```diff
@@ -1,5 +1,12 @@
+import { Team } from './Team'
+
 export interface PlayerRanking {
-  readonly name: string
   readonly id?: number
+  readonly name: string
+  readonly country?: string
+  readonly teams?: Team[]
+  readonly maps?: number
+  readonly kdDiff?: number
+  readonly kd?: number
```

Comment:
Same for there. They're always there so they should not be optional.

Reply:
Removed the optional hints for my changes. I left the id to optional as this was not my change so I am not sure if there is an edge case where the ID might be empty.. ?